Makefile

```diff
@@ -164,9 +168,6 @@ build:
 	GOBUILD_FLAGS="$(GOBUILD_FLAGS)" \
 	build/go-in-docker.sh build/build.sh
 
-	@echo "+ Copying artifact to temporary directory"
```

Comment:
I figured this didn't make sense here, in isolation, because `make build` and `make container` could not be invoked separately.